diffusion/models/models.py

```diff
@@ -310,6 +332,11 @@ def stable_diffusion_xl(
                 layer = zero_module(layer)
         # Last conv block out projection
         unet.conv_out = zero_module(unet.conv_out)
+    if type(latent_mean) is float:
```

Comment:
Technically should use isinstance of here

Reply:
Fixed!